shared/uavobjectdefinition/stabilizationsettings.xml

```diff
@@ -52,8 +52,8 @@
 		<field name="DeadbandSlope" units="%" type="uint8" elementnames="Roll,Pitch,Yaw" defaultvalue="60,60,50" limits="%BE:0:100,%BE:0:100,%BE:0:100">
 			<description>Sets the slope of the deadband area in the PID controller.</description>
 		</field>
-		<field name="RCControlSmoothing" units="" type="enum" elementnames="Axes,Thrust" options="None,Normal,Extended" defaultvalue="None">
-			<description>Enables different ways of input signal smoothing, to reduce excessive P- and D-term excitation in the PID controller. Normal and Extended modes chamfer the leading edges in combination with some amount of prediction, to avoid delay.</description>
+		<field name="RCControlSmoothing" units="" type="enum" elementnames="Axes,Thrust" options="None,Normal,Extended,Linear,Psychic" defaultvalue="Normal">
```

Comment:
Shouldn't this be ManualControlSmoothing?

Reply:
There wasn't any opposition to the current name when it first was committed. But changing it should be OK, if ManualControlSmoothing sounds better and is more in line with naming. The setting is basically unused currently, apart from me, and renaming it shouldn't break things on upgrades I'd figure.

Reply:
It's a usability issue to use different terminology for the same thing in user-facing stuff like UAVOs.